dll/win32/imm32/imm.c

```diff
@@ -171,7 +171,7 @@ BOOL WINAPI ImmFreeLayout(DWORD dwUnknown)
     }
     else
     {
-        hNewKL = (HKL)(DWORD_PTR)dwUnknown;
+        hNewKL = (HKL)UlongToHandle(dwUnknown);
```

Comment:
Is the HKL cast still needed? (here and in the other changes below and in the other files)

Reply:
Deleted.